src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -1564,7 +1611,7 @@ static int xgq_freq_scaling_by_topo(struct platform_device *pdev,
 	int system_clk_count = 0;
 	int clock_type_count = 0;
 	unsigned short target_freqs[4] = {0};
-	int i = 0;
+	int i = 0, ret = 0;
```

Comment:
There is a lint error. The ret is not being used anymore.

Please run
make clean;make -j

we should get a cleaner result without warnings. If you prefer, please fix those too.
if you prefer not to do so, you can ignore other developers's code. We will clean those up later.